src/Panel/Assets.php

```diff
@@ -331,14 +337,17 @@ public function url(): string
 	 * Get the correct Vue script URL depending on dev mode
 	 * and the enabled/disabled template compiler
 	 */
-	public function vue(bool $production = true): string
+	public function vue(): string
 	{
-		$script = $this->kirby->option('panel.vue.compiler', true) === true ? 'vue' : 'vue.runtime';
+		// During dev mode, load the dev version of Vue
+		if ($this->isDev === true) {
+			return $this->url . '/node_modules/vue/dist/vue.esm.browser.js';
```

Comment:
In dev mode, we now always load the full Vue bundle. Before, if the `panel.vue.compiler` was set to `false` it would load a runtime-only bundle. But I think dev mode should always override this config option.

Reply:
I'm not sure about this to be honest. When you intentionally need to switch off the compiler in a project, but it still works in dev environments, it might cause quite some confusion.

Reply:
But who actually runs develop mode in a project? This doesn't mean debug mode, it's `panel.dev` config active AND vite running in `npm run dev`.

Reply:
🙈 totally mixed it up. You are right. In this case, it makes perfect sense.